### ISSUE TITLE
Incorrect normalisation with torch tensor input

### ISSUE THREAD

**Issue by montardon** [ultralytics/ultralytics#6811]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Predict

### Bug

I'm using a trained model to make predictions, for object detection.
My code looks like:
```
images = loaded images
print(f'Images shape : {images.shape}') # outputs (1, 480, 640, 3)

model = YOLO(model_dir_path)
tensor = torch.tensor(images.transpose(0, 3, 1, 2))
# tensor is a batch of size (N,C,H,W)
print(f'Tensor shape {tensor.shape}') # outputs torch.Size([1, 3, 480, 640])

inferences = model(source=tensor)

 for (i,pred_tmp) in enumerate(inferences):
    for n in range(0,len(pred_tmp.boxes.conf)):
            print(f'Original img size {pred_tmp.orig_img.shape}') # outputs torch.Size([1, 3, 480, 640])
            print(f'Original shape {pred_tmp.orig_shape}')  # torch.Size([1, 3])    <---  looks strange 
            print(f'pred_tmp.boxes.xyxy {pred_tmp.boxes.xyxy}') # tensor([[365.5303, 147.1351, 553.0533, 305.4365]])
            print(f'pred_tmp.boxes.xyxyn {pred_tmp.boxes.xyxyn}') # tensor([[121.8434, 147.1351, 184.3511, 305.4365]])
```
I was expecting an orig_shape of size (480,640). So xyxyn does not have the right values.

### Environment

Ultralytics YOLOv8.0.187 🚀 Python-3.9.13 torch-2.0.1+cu117 CUDA:0 (NVIDIA A100 80GB PCIe, 80995MiB)
Setup complete ✅ (128 CPUs, 503.4 GB RAM, 21.4/888.3 GB disk)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by montardon** [comment]
Updating to a newer version solved the problem.

**Comment by glenn-jocher** [comment]
@montardon that's great to hear! Always feel free to reach out if you encounter any more issues. Happy detecting! 😊
